internal/node/component/info/network_config.go

```diff
@@ -63,6 +64,33 @@ type workerConfig struct {
 }
 
 var defaultNetworkParameters = map[network.Source]*Parameters{
+	network.ActivityPubSource: {
+		KafkaTopic: &ConfigDetail{
+			IsRequired:  true,
+			Type:        StringType,
+			Value:       "activitypub_events",
+			Description: "The Kafka topic to publish the ActivityPub data to. If not provided, the data will be published to the default topic",
```

Comment:
"The Kafka topic to which the ActivityPub data will be published. By default, the data will be sent to this topic on the Kafka broker running on the Mastodon instance."